pillar.example

```diff
@@ -45,24 +45,27 @@ postgresql:
     shared_buffers: 128MB
     maintenance_work_mem: 64MB
     effective_cache_size: 4GB
+  
+  # These lists will be written in the pg_hba.conf
+  # Accepted shape:   
+  #  - ['host/local', 'DATABASE',  'USER',  'ADDRESS', 'METHOD']
+
+  acls:
+    - ['host', 'triebwerk', 'deploy', '10.14.38.0/24', 'trust']
+    - ['host', 'triebwerk', 'user_wwith_password', '10.14.38.0/24', 'password']
+    - ['host', 'repliction', 'replicant', '10.14.38.0/24', 'trust']
 
-  # These users will be created and added to pg_hba.conf
   users:
     - username: deploy
-      type: host
-      database: triebwerk
-      address: 10.14.38.0/24
-      method: trust
     - username: user_with_password
-      type: host
-      database: triebwerk
-      address: 10.14.38.0/24
-      method: trust
       options: PASSWORD 'supersecret' NOSUPERUSER NOCREATEDB NOCREATEROLE INHERIT LOGIN
     - username: replicant
-      type: host
-      database: replication
-      address: 10.14.38.0/24
-      method: trust
       options: NOSUPERUSER NOCREATEDB NOCREATEROLE INHERIT LOGIN REPLICATION
-      password:
+
+  databases:
+    - database: triebwerk
+      owner: deploy
+      schemas:
+      - schemaname: test
+        user: user_with_password
+      extensons: [postgis]
```

Comment:
This looks pretty straight forward!
Is this actually a usecase to have a different schema user than the database owner?

What do you think of renaming the schema `user` attribute with `authorization` (matching the SQL `AUTHORIZATION` keyword)?

Reply:
Yes, then I have a better control over which user can access what resource, instead of handing over the DB owner to all our users.

Reply:
```yaml
  # NOTE: Database and user need to be existent
  schemas:
    - database: triebwerk
      schema: a
      authorization: postgres
```